movie-app/src/scss/components/body/_search_results.scss

```diff
@@ -1,57 +1,53 @@
 .search-container {
-    width: 100vw;
-    display: grid;
-    margin-top: 20px;
-    margin-bottom: 20px;
-    padding-left: 5%;
-    padding-right:5%;
-    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
-    grid-column-gap: 20px;
-    grid-row-gap: 20px;
+  width: 100vw;
+  display: grid;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding-left: 5%;
+  padding-right: 5%;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-column-gap: 20px;
+  grid-row-gap: 20px;
```

Comment:
CSS takes a long time to make it look beautiful, but this looks like a masterpiece